### PR TITLE
Add bio for Javier Albornoz

### LINKED ISSUE #17: Add bio for ajavier
- [x] Create a new branch
- [x] Add bui file for ajavier
- [x] Create a pull request
- [x] Get feedback
- [x] Merge the pull request

### PR DESCRIPTION
**Pull request by ajavier**
Closes #17 

Short personal info for the github user @ajavier.

### CODE REVIEW

File: ajavier.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Javier Albornoz
+
+* Name: Javier Albornoz
+* Location: Heredia, Costa Rica
+* Username: @ajavier
```
Comment (ajavier) [inline ajavier.md]: Can you include your favorite movie?

Comment (johnclc) [inline ajavier.md:5]: @ajavier - can you add your current time where you are?

**Comment by ajavier** [comment]
Closes #17 

What do you think @johnclc?

**Comment by johnclc** [comment]
Awesome! :+1:

**Pull request merged**